.weave/migrations/slack/env.py

```
@@ -1,7 +1,8 @@
 import os
 import sys
 from logging.config import fileConfig
-from sqlalchemy import engine_from_config, pool
+from sqlalchemy import engine_from_config, pool, create_engine, text
+from sqlalchemy.exc import OperationalError
```

Ruff (0.11.9)

`sqlalchemy.exc.OperationalError` imported but unused

Remove unused import: `sqlalchemy.exc.OperationalError`

(F401)